src/lib/components/MediaObject/MediaObject.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './MediaObject.css';
+
+const MediaObject = (props) => {
+  let image;
+  let title;
+  let description;
+  let metadata;
+
+  if (props.img.src) {
+    image = (
+      <img
+        className={`p-media-object__image${props.round ? ' is-round' : ''}`}
+        src={props.img.src}
+        alt={props.img.alt}
+      />
+    );
+  }
+
+  if (props.title.name) {
+    if (props.title.link) {
+      title = (
+        <h3 className="p-media-object__title">
+          <a href={props.title.link}>{props.title.name}</a>
+        </h3>
+      );
+    } else {
+      title = (
+        <h3 className="p-media-object__title">
+          {props.title.name}
+        </h3>
+      );
+    }
+  }
+
+  if (props.description) {
+    description = (
+      <p className="p-media-object__content">
+        {props.description}
+      </p>
+    );
+  }
+
+  if (props.metadata) {
+    const metadataItems = Object.keys(props.metadata).map((key) => {
+      if (key === 'topic') {
+        return (
+          <li key={key} className="p-media-object__meta-list-item">
+            {props.metadata[key]}
+          </li>
+        );
+      }
+      return (
+        <li key={key} className={`p-media-object__meta-list-item--${key}`}>
+          {props.metadata[key]}
+        </li>
+      );
+    });
+
+    metadata = (
+      <ul className="p-media-object__meta-list">
+        {metadataItems}
+      </ul>
+    );
+  }
+
+  return (
+    <div className={props.large ? 'p-media-object--large' : 'p-media-object'}>
+      {image}
+      <div className="p-media-object__details">
+        {title}
+        {description}
+        {metadata}
+      </div>
+    </div>
+  );
+};
+
+MediaObject.defaultProps = {
+  round: false,
+  large: false,
+  img: {
+    src: '',
+    alt: '',
+  },
+  title: {
+    name: '',
+    link: null,
+  },
+  description: null,
+  metadata: null,
```

Comment:
you could do like you to for title:
```
metadata: {
    topic:  '',
    date:  '',
    venue:  '',
    location: '',
  }
```

Reply:
If I do this, the metadata section is rendered in the DOM unnecessarily because it's treating the object as an actual thing that exists (which I don't want)

Reply:
Good point. Nothing is actually required so I'll change the other default props to null